docs/components/alert/examples/FwbAlertExampleCustomContent.vue

```
@@ -23,36 +23,41 @@
             Dismiss
-          </button>
+          </fwb-button>
         </div>
       </template>
     </fwb-alert>
   </div>
 </template>
 
 <script lang="ts" setup>
-import { FwbAlert } from '../../../../src/index'
+import { FwbAlert, FwbButton } from '../../../../src/index';
```

ESLint

[error] Extra semicolon.

(@stylistic/semi)